multicast/hear.py

```
@@ -829,15 +829,15 @@ def doStep(self, *args, **kwargs):
 				server.serve_forever()
-		except KeyboardInterrupt as userInterrupt:
+		except KeyboardInterrupt as _cause:
 			try:
 				if server and server.socket:  # pragma: no cover
 					old_sock = server.socket
 					multicast.endSocket(old_sock)
 			finally:
 				raise KeyboardInterrupt(
 					f"HEAR has stopped due to interruption signal (was previously listening on ({HOST}, {PORT}))."
-				) from userInterrupt
+				) from _cause
```

Ruff (0.8.2)

Avoid specifying long messages outside the exception class

(TRY003)